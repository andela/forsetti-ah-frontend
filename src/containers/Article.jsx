import React, { Component } from 'react';
import { SingleArticle } from '../components/GetSingleArticle';

class Article extends Component {
  render() {
    return (
      <SingleArticle />
    );
  }
}

export default Article;
