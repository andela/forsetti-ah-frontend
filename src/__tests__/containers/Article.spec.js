import React from 'react';
import { shallow } from 'enzyme';
import Article from '../../containers/Article';

describe('Article Container', () => {
  test('should render Article container container', () => {
    const container = shallow(<Article />);

    expect(container).toMatchSnapshot();
  });
});
