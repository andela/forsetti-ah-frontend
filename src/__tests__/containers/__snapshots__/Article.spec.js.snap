// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article Container should render Article container container 1`] = `ShallowWrapper {}`;
